src/components/SortableList/SortableList.tsx

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import { GoodsList } from '../GoodsList';
+
+import './SortableList.scss';
+
+type Props = {
+  goods: string[];
+};
+
+enum SortOrder {
+  none = 0,
+  alphabetic = 1,
+  length = 2,
+}
```

Comment:
це і так значення заумовчуванням)

```suggestion
  none,
  alphabetic,
  length,
}
```